apps/studio/src/lib/editor/vim.ts

```diff
@@ -18,8 +20,19 @@ export function applyConfig(codeMirrorVimInstance: any, config: Config) {
   }
 }
 
-export function setKeybindingsFromVimrc(codeMirrorVimInstance: any) {
-  const potentialCommands = window.main.readVimrc();
+async function readVimrc(pathToVimrc?: string): Promise<string[]> {
+  const userDirectory = window.main.platformInfo().userDirectory;
```

Comment:
Make sure you merge in main and then this will need to change. PlatformInfo is now available on `window.platformInfo` so you should be able to change this to `window.platformInfo.userDirectory`